lib/http-tequilapi-client.js.flow

```diff
@@ -19,12 +19,14 @@ import { NodeHealthcheckDTO } from "./dto/node-healthcheck";
 import { ProposalDTO } from "./dto/proposal";
 import { ConnectionRequest } from "./dto/query/connection-request";
 import { ProposalQueryOptions } from "./dto/query/proposals-query-options";
-import { SessionDTO } from "./dto/session";
-declare export class HttpTequilapiClient implements TequilapiClient {
+import { ServiceInfoDTO } from "./dto/service-info";
+import { ServiceRequest } from "./dto/service-request";
+declare export class HttpTequilapiClient mixins TequilapiClient {
```

Comment:
Please fix Flow definitions - `flowgen` is failing sometimes, so you have to check in `master`, where it is already fixed:
```suggestion
declare export class HttpTequilapiClient implements TequilapiClient {
```

Reply:
?? @donce are you aware of the issues this could cause? Can't find anything meaningful in the docs. As far as I can tell this should work too.

Reply:
The only issue is that if `TequilapiClient` extends another interface, `HttpTequilapiClient` won't inherit those props/methods.

Reply:
Oh, ok, I can see that the build is failing because of this. @Waldz

Reply:
Exactly.